qt/scientific_interfaces/Muon/ALCDataLoadingPresenter.cpp

```diff
@@ -135,6 +140,7 @@ void ALCDataLoadingPresenter::loadFilesCurrentlyInModel() {
   } catch (std::exception &e) {
     throw std::runtime_error(e.what()); // Caught in handle load request
   }
+  m_subscriber->loadedDataChanged();
```

Comment:
Because `m_subscriber` is initially null, it would be wise to do a null check before calling `loadedDataChanged` on that.

Reply:
Using an `assert` sounds like a great idea since `m_subscriber` is analogous to the car engine in this use case!